### PR TITLE
Adding Converter cli

### PR DESCRIPTION
**Pull request by minhtuev**
`mindmeld convert --rs rasa_project`
`mindmeld convert --df dialogflow_project`

### CODE REVIEW

File: mindmeld/cli.py [hunk 1]
```diff
@@ -445,6 +446,36 @@ def setup_blueprint(ctx, es_host, skip_kb, blueprint_name, app_path):
...
+@click.pass_context
+@click.option('-d', '--df', is_flag=True, help="Convert a Dialogflow project")
+@click.option('-r', '--rs', is_flag=True, help="Convert a Rasa project")
+@click.argument('project_path', required=True, type=click.Path(exists=True))
+@click.argument('mindmeld_path', required=False)
+def convert(ctx, df, rs, project_path, mindmeld_path=None):
+    """Converts a Rasa or DialogueFlow project to a MindMeld project"""
+    try:
+        project_path = os.path.abspath(project_path)
+        mindmeld_path = os.path.abspath(mindmeld_path or "converted_app")
```
Comment (vijay120) [inline mindmeld/cli.py]: Why is `converted_app ` here?

Comment (minhtuev) [reply mindmeld/cli.py]: To give a default location for the converted app in case the user does not specify

Comment (vijay120) [reply mindmeld/cli.py]: Got it, please fix the linting issues and merge.

File: mindmeld/cli.py [hunk 2]
```diff
@@ -445,6 +446,36 @@ def setup_blueprint(ctx, es_host, skip_kb, blueprint_name, app_path):
...
+        converter_cls = {'Rasa': RasaConverter, 'Dialogflow': DialogflowConverter}
+        if df:
+            framework = 'Dialogflow'
+        elif rs:
+            framework = 'Rasa'
+        else:
+            logger.warning("Please specify the project's platform Rasa/Dialogflow.")
```
Comment (serapio) [inline mindmeld/cli.py]: "Please specify the project's platform type (Rasa/Dialogflow)."

You can do this arg check outside the try block.

File: mindmeld/cli.py [hunk 3]
```diff
@@ -445,6 +446,36 @@ def setup_blueprint(ctx, es_host, skip_kb, blueprint_name, app_path):
...
+            ctx.exit(1)
+
+        converter_cls = converter_cls[framework]
+        converter = converter_cls(project_path, mindmeld_path)
+        converter.convert_project()
+        logger.info(f"Successfully converted {framework} project at {project_path} to"
+                    f" MindMeld project at {mindmeld_path}.")
```
Comment (serapio) [inline mindmeld/cli.py]: ☠️

**Pull request merged**